### DISCUSSION TITLE
[Help]: Custom Scrollbars

### DISCUSSION THREAD

**Discussion by dante-blitz** [davidjerleke/embla-carousel#803] [Help] [answered]
### Summary

Have tried to get a horizontal embla carousel working with various 3rd parties custom scrollbar packges e.g. https://www.npmjs.com/package/overlayscrollbars-react

Not having much joy. The custom scrollbars don't work.

Has anybody go embla caorusel working with custom scrollbars?

### If applicable, which variants of Embla Carousel are relevant to this question?

- [ ] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment]
@dante-blitz there's no example demonstrating this yet but I will try to build it when I get the chance.

**Reply by davidjerleke** [reply, marked as answer]
@dante-blitz is this example what you're looking for?

- https://github.com/davidjerleke/embla-carousel/discussions/1030#discussioncomment-13340084
